internal/server/server.go

```diff
@@ -132,8 +148,13 @@ func FromCluster(in *redskyv1beta1.Experiment) (redskyapi.ExperimentName, *redsk
 		})
 	}
 
+	// We need all of the assignments to include the baseline
+	if len(baseline.Assignments) == 0 || len(baseline.Assignments) != len(out.Parameters) {
```

Comment:
Should we surface an error here if `len(baseline.Assignments) != len(out.Parameters)`?

Reply:
I was just silently ignoring it and treating it as a nice to have. Maybe I can add something to `check experiment`...